rollup.config.js

```diff
@@ -162,9 +162,13 @@ export default function (args) {
   let c = Object.keys(args).find((key) => key.startsWith('config-'))
   if (c) {
     c = c.slice('config-'.length).replace(/_/g, '/')
+    if (c === 'macro-vite') {
```

Comment:
Doesn't `config-macro_vite` work??

Reply:
I'll try that (I'm having some issues with types/exports, when importing valtio/macro/vite)

Reply:
I mean L164 does the conversion, no?

Reply:
oh, i thougt you mean the cli command? yea, it does! in the end, it's valtio/macro/vite.js

Reply:
Please remove L165-L167,

Reply:
remove L171 empty line to to remove the changes in this file.